Source/iop/Iop_Spu2.h

```diff
@@ -1,13 +1,13 @@
 #ifndef _SPU2_H_
 #define _SPU2_H_
 
-#include <boost/utility.hpp>
+#include "Noncopyable.h"
 #include <functional>
 #include "Iop_Spu2_Core.h"
 
 namespace Iop
 {
-	class CSpu2 : boost::noncopyable
+	class CSpu2 : Framework::CNoncopyable

```

Comment:
The noncopyable thing was useful back when C++11 was not widely available. IMO, now, the proper way of carrying this intention would be to mark the copy constructor and copy assignment operator as deleted. ex: `CSpu2(const CSpu2&) = delete;`

No need for a Noncopyable utility in Framework.

Reply:
ah, ill change it so